src/app/action-items/action-items.component.ts

```diff
@@ -63,6 +64,7 @@ export class ActionItemsComponent implements OnInit, OnDestroy {
           this.retrieveActionItems();
           this.justAddedItemId = item.id;
           this.router.navigate(['/items'], { queryParams: { id: this.justAddedItemId } });
+          this.scrollToLastItem = true;
```

Comment:
**FIX:** This can't be the last item, as if the application is used by multiple users then at the same time another user may add an action item and after refreshing the list this item will be the last one. You'd need to use id.